tests/sparseml/onnx/optim/test_sensitivity_ks.py

```diff
@@ -203,9 +203,12 @@ def test_approx_ks_loss_sensitivity(
         onnx_models_with_analysis.loss_approx_path
     )
     expected_layers = sorted(
-        expected_analysis.dict()["results"], key=lambda x: x["index"]
+        expected_analysis.dict()["results"],
+        key=lambda x: (x["id"], x["index"], x["name"]),
```

Comment:
why move `id` to the first sort key, instead of `index` as we have now with `id` as the tiebreaker?

Reply:
id seemed more of a unique thing we should be sorting on. I can change back to index first